org.aposin.licensescout.core/src/test/java/org/aposin/licensescout/util/JsonUtilTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright ${year} ${owner}
```

Comment:
Ups, sorry for this, but I did not realize that the license header contains placeholders for the year and owner. This should be the following:

```suggestion
 * Copyright 2019 Association for the promotion of open-source insurance software and for the establishment of open interface standards in the insurance industry (Verein zur Förderung quelloffener Versicherungssoftware und Etablierung offener Schnittstellenstandards in der Versicherungsbranche)
```

Reply:
Sorry @shreenelakurthi, I don't have access to your branch to commit the changes. Please, press the "commit suggestion" button and we will be ready to merge this into the project. Thanks again and sorry for any inconvenience.